### PR TITLE
HDDS-4189. Change `ozone admin om getserviceroles` to `ozone admin om roles`

### PR DESCRIPTION
**Pull request by amaliujia**
## What changes were proposed in this pull request?

Change `ozone admin om getserviceroles` to `ozone admin om roles`

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4189

## How was this patch tested?

Unit Test

### CODE REVIEW

**Comment by amaliujia** [comment]
R: @adoroszlai @timmylicheng

**Comment by amaliujia** [comment]
Context: https://github.com/apache/hadoop-ozone/pull/1346#discussion_r481380452

**Review by cxorm — APPROVED** [review]
Thanks @amaliujia for the work.

This modification looks good to me.

**Comment by amaliujia** [comment]
Thank you for your review @cxorm!

```
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.417 s <<< FAILURE! - in org.apache.hadoop.ozone.om.request.key.TestOMAllocateBlockRequest
[ERROR] testValidateAndUpdateCache(org.apache.hadoop.ozone.om.request.key.TestOMAllocateBlockRequest)  Time elapsed: 0.114 s  <<< FAILURE!
java.lang.AssertionError: Values should be different. Actual: 1599003025036
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failEquals(Assert.java:185)
	at org.junit.Assert.assertNotEquals(Assert.java:161)
	at org.junit.Assert.assertNotEquals(Assert.java:198)
	at org.junit.Assert.assertNotEquals(Assert.java:209)
	at org.apache.hadoop.ozone.om.request.key.TestOMAllocateBlockRequest.testValidateAndUpdateCache(TestOMAllocateBlockRequest.java:100)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)

```

The failed UT seems not related to this change.

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java [hunk 1]
```diff
@@ -30,10 +30,10 @@
 import java.util.concurrent.Callable;
 
 /**
- * Handler of om get-service-roles command.
+ * Handler of om status command.
  */
 @CommandLine.Command(
-    name = "getserviceroles",
+    name = "status",
```
Comment (adoroszlai) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java:36]: I think we should keep `getserviceroles` as an alias for compatibility.

```suggestion
    name = "status", aliases = "getserviceroles",
```

**Review by adoroszlai — CHANGES_REQUESTED** [review]
Thanks @amaliujia for working on this.  Can you please add an alias to avoid breaking any existing scripts that might use this command?

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java (hunk 1 above)
Comment (cxorm) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java:36]: Thanks @adoroszlai for the idea, I agree with it.

Comment (amaliujia) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java]: That makes sense. Suggestion applied.

**Comment by amaliujia** [comment]
It seemed that there is another opinion about what command name should be. So I will convert this PR to a draft now to wait for a consensus. 

Thanks for all reviews so far and sorry for the confusion.

**Comment by amaliujia** [comment]
@adoroszlai 

Ok finally I think we can use `roles` based on https://github.com/apache/hadoop-ozone/pull/1346#discussion_r481746270.

I made necessary change in this PR. Can you take another look?

**Review by adoroszlai — APPROVED** [review]
Thanks @amaliujia for this improvement.

**Pull request merged**